textbook/2_grouping.qmd

```diff
@@ -0,0 +1,328 @@
+---
+title: "Grouping"
+include-in-header:
+  - text: |
+      <style>
+        .table.dataframe {
+          width: auto;
+          text-align: right;
+        }
+      </style>
+fig-width: 4
+fig-height: 4
+resources:
+  - data
+  - utils.py
+---
+
+::: {.callout-note title="Learning outcomes"}
+- **Explain** how grouping allows you to directly compare categories and helps you to avoid making incorrectly generalized conclusions (e.g., Simpson’s Paradox).
+
+- **Apply** grouping strategies using color, size, shape and position to visually distinguish between categories via `.encode(color='category')` and `aes(color=category)`.
```

Comment:
One thought I had for LOs in general, is that how do we include code if more than two packages are used in the book... although that can be a future problem as I'm not sure how likely it is to happen and I don't think it is worth removing  the code just for that reason